### ISSUE TITLE
save the window state 

### ISSUE THREAD

**Issue by zhangmingze** [azain/electron-window-manager#15]
to save the window state when restart

**Comment by shrimpwagon** [comment]
Can you elaborate a bit? Do you mean open in same position and size?

**Comment by zhangmingze** [comment]
yeah.like [this](https://github.com/Sethorax/electron-window-state-manager)
May be can provide some options on ```createNew(...)``` or ```open(...)```,so we don't need to use another lib.

**Comment by shrimpwagon** [comment]
That looks pretty cool. Can you do it?

**Comment by dannyrb** [comment]
We've accomplished this in our project by listening for `BrowserWindow` events and updating a "LastUsedConfiguration" when those events fire: move, resize, etc.

When the app starts up, it uses that config to restore the previous window state.
